pkg/kdump/kdump-view.jsx

```diff
@@ -324,6 +382,26 @@ export class KdumpPage extends React.Component {
                                          handleSave={this.props.onSaveSettings} />);
     }
 
+    handleAutomationClick() {
+        const Dialogs = this.context;
+        const shell = `
+# A reboot will be required if crashkernel was not set before
+kdumpctl reset-crashkernel
+systemctl enable --now kdump.service
+cat > /etc/kdump.conf << EOF
+${this.props.exportConfig(this.props.kdumpStatus.config)}
```

Comment:
This needs some shell escaping, at least of any `$` and `|`

Reply:
As far as I can see only `$` is really needed, `|` from some experiments seems mostly harmless?